src/components/OSCALControlPart.js

```diff
@@ -32,7 +32,13 @@ export default function OSCALControlPart(props) {
   }
 
   if (props.part.name === "guidance") {
-    return <OSCALControlGuidance prose={props.part.prose} />;
+    return (
+      <OSCALControlGuidance
+        prose={props.part.prose}
+        id={props.control.id}
```

Comment:
To be clear about test failure, `id` is required. https://pages.nist.gov/OSCAL/reference/latest/complete/json-reference/#/catalog/controls

That looks like it may be an issue with our tests. Could you look into adding an id?